### PR TITLE
[378821] Add option to generate main method

### PR DESCRIPTION
**Pull request by kthoms**
Option added like in JDT's class wizard:

screenshot 34

Created file:
screenshot 35

Signed-off-by: Karsten Thoms karsten.thoms@itemis.de

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=378821

File: org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/wizards/XtendTypeCreatorUtil.java [hunk 1]
```diff
@@ -90,6 +91,15 @@ public static String createClassContent(String typeName, String superClass,
 		sb.append(" {");
 		sb.append(lineSeparator);
 		sb.append(indentation);
+		if (createMain) {
+			sb.append("def static void main (String[] args) {");
```
Comment (cdietrich) [inline org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/wizards/XtendTypeCreatorUtil.java]: is the space between main and () intended? the formatter does not add it

**Comment by kthoms** [comment]
You are right, the formatter would not add the space. So the wizard shouldn't do that also.
Changed.

**Comment by cdietrich** [comment]
thanks

**Pull request merged**